content/guides/set-up-rails-appplication-with-omniauth-bookingsync.md

```diff
@@ -0,0 +1,303 @@
+# Building New Rails Application With Devise And BookingSync Integration
+
+If you need to integrate your existing Rails application (which most likely uses Devise for authentication) with BookingSync, that's the exact guide you are looking for.
+
+## Setup
+
+After setting up environment for Ruby (check either [rvm](https://rvm.io) or [rbenv](https://github.com/rbenv/rbenv)), install `bundler` and `rails` gems:
+
+~~~
+gem install bundler
+gem install rails
+~~~
+
+And generate new Rails application:
+
+~~~
+rails new omniauth-bookingsync-app
+cd omniauth-bookingsync-app
+~~~
+
+To finish the setup we just need to create the database:
+
+~~~
+bundle exec rake db:create
+~~~
+
+To verify it works, just run:
+
+~~~
+rails server
+~~~
+
+And visit `http://localhost:3000`.
+
+
+## Adding Devise
+
+Add [`devise`](https://github.com/plataformatec/devise) gem to Gemfile:
+
+~~~ Gemfile
+gem 'devise'
+~~~
+
+And run:
+
+~~~
+bundle install
+~~~
+
+Then run the generator:
+
+~~~
+rails generate devise:install
+rails generate devise:views
+~~~
+
+We don't need to have a signup / signin alternative with BookingSync, just a way to integrate the account, so we can remove this part from `app/views/devise/shared/_links.html.erb`:
+
+~~~ html app/views/devise/shared/_links.html.erb
+<%- if devise_mapping.omniauthable? %>
+  <%- resource_class.omniauth_providers.each do |provider| %>
+    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
+  <%- end -%>
+<% end -%>
+~~~
+
+Another step would be generating a model for Devise, for the sake of simplicity let's call it `Account` to match the concept from BookingSync authentication resource:
+
+~~~
+rails generate devise account
+~~~
+
+The default migration generated by that command should be sufficient for this example app. Now you can run the migration:
+
+~~~
+rake db:migrate
+~~~
+
+## Adding OmniAuth and integrating BookingSync
+
+Go to the [BookingSync Partners section](https://www.bookingsync.com/en/partners/login) and register you account or sign in if you already have one. After you sign in, create a new application. You will need to the provide a name, an admin URL (the URL to which the user should be redirected to after signing in when developing `embedded` application) and a redirect URI. Here's an example:
+
+<div>
+  <img alt="BookingSync" src="/images/bookingsync_new_application.png">
+</div>
+
+In this case we want to have a separate application, not the one that should be accessible only from the applications' store, so `standalone` application is the way to go.
```

Comment:
Why would you want to have embedded application? If it's separate from BookingSync ecosystem, you would probably want to have it standalone instead of embedding, no? Or maybe I don't get the use case, but can't really think why it would be better to have embedded application here.